web/src/components/MapView.tsx

```diff
@@ -86,26 +71,27 @@ const MapView = React.forwardRef((props: MapViewProps, ref: React.Ref<MapRef>):
 
   const { viewportSmall } = useWindowDimensions()
 
-  const onDeselectFeature = useCallback(() => {
-    setQueryLocation(null)
-    selectFeature(null)
-    updateQueryParams()
-  }, [selectFeature, setQueryLocation])
+  const onDeselectFeature = useCallback(
+    e => {
+      // Currently selected feature should not be deselected if the user clicks on the controls like zoom or user location
+      if (!e.target.className.includes('mapboxgl-ctrl-icon')) {
```

Comment:
```suggestion
      if (e.target.className.includes('mapboxgl-canvas')) {
```
i think this is slightly better, because if we place f.e. another button or input field on the map, we would have to add classes to your includes fkt

Reply:
Good idea, will do in #464